docs/customization/index.rst

```diff
@@ -91,6 +91,39 @@ Ansible configuration
 
 By default, Enos loads its own ``ansible.cfg``. To use another Ansible
 configuration file, the ``ANSIBLE_CONFIG`` environment variable can be used.
-Further information can be found : `see here 
+Further information can be found : `see here
 <http://docs.ansible.com/ansible/intro_configuration.html>`_.
 
+
+Using a persistent registry with Ceph
+-------------------------------------
+
+To save time during deployment, you can setup a Ceph Rados Block
```

Comment:
Enos deploys a fresh registry that acts as a private docker registry mirroring the official one and cache containers close to your deployment resources.

To get a persistent registry you can use  a persistent Ceph Rados Block Device for the registry backend. Image will be cached during the first deployment and reused for the subsequent deployments

The relevant configuration ...